hypershift-operator/controllers/nodepool/nodepool_controller.go

```diff
@@ -1303,6 +1308,17 @@ func (r *NodePoolReconciler) getConfig(ctx context.Context,
 		configs = append(configs, *configConfigMap)
 	}
 
+	// Look for operator generated MachineConfigs from the hosted control plane namespace
```

Comment:
nit: look for nto generated...
Might seems obvious but people with less context would wonder

Reply:
Thanks, I fixed this comment.